### ISSUE TITLE
Cannot maintain zoom while updating series or options

### ISSUE THREAD

**Issue by montekarlos** [apexcharts/Blazor-ApexCharts#161]
G'day

Very happy with Blazor-ApexCharts so far!

The one issue I've hit is that I would like to preserve zoom when updating series / options. If I have code like

```
await _chart!.UpdateOptionsAsync(false, false, false);
await _chart!.ZoomXAsync(_xZoomMin, _xZoomMax);
```

I can see the chart refresh after UpdateOptionsAsync (drawing the full series) and then again after ZoomXAsync (drawing the zoomed series) which produces an ugly visual flicker

This PR: https://github.com/apexcharts/Blazor-ApexCharts/pull/160 adds a new call `UpdateOptionsWithZoomAsync`

```
await _chart!.UpdateOptionsWithZoomAsync(false, false, false, _xZoomMin, _xZoomMax);
```

This does not cause flickering for me as the zoom happens before the refresh

Note that I have animations disabled

I'm open to a better solution to this problem!

**Comment by joadan** [comment]
Hello,

Thank you for your contribution!
Related: https://github.com/apexcharts/apexcharts.js/issues/643 

I'll look into your PR

**Comment by montekarlos** [comment]
Thank joadan that looks great! I'll close my PR

**Comment by joadan** [comment]
Fixed in v0.9.15-beta
